cmd/kops/integration_test.go

```diff
@@ -338,20 +344,35 @@ func runTestAWS(t *testing.T, clusterName string, srcDir string, version string,
 		expectedFilenames = append(expectedFilenames, s)
 	}
 
-	if private {
+	manageIAM := true
+	for _, override := range lifecycleOverrides {
+		if override == "IAMRole=Ignore" {
```

Comment:
These should be a list of

Task1=Ignore,Task2=Ignore.

You will need to split the sting on commas.